From: Dominic Meiser <alpine@msrd0.de>
Date: Tue, 22 Dec 2020 23:31:28 +0100
Subject: [PATCH] Do not install libunwind source

This was added in rustc 1.48, but is incompatible with the Alpine Linux package since it removes all bundled
dependencies prior to building.

--- a/src/bootstrap/dist.rs
+++ b/src/bootstrap/dist.rs
@@ -889,7 +889,7 @@ impl Step for Src {
         copy_src_dirs(
             builder,
             &builder.src,
-            &["library", "src/llvm-project/libunwind"],
+            &["library"],
             &[
                 // not needed and contains symlinks which rustup currently
                 // chokes on when unpacking.
